docker/Makefile

```diff
@@ -0,0 +1,32 @@
+include .env
+
+COMMIT_SHA ?= $(shell git rev-parse --short HEAD)
+AWS_REGION ?= $(shell cat ~/.aws/config | grep -m 1 region | sed 's/region = //')
+AWS_ACCOUNT_ID ?= $(shell aws sts get-caller-identity --query "Account" --output text)
+DOCKER_REGISTRY_URL ?= ${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_REGION}.amazonaws.com
+
+C_GREEN ?= \033[32m
+C_RESET ?= \e[0m
```

Comment:
Great work Alan, everything worked just fine. Just a very small detail.
I think the reset variable should be something like `C_RESET ?= \033[0m`, shouldn't it?

Reply:
Thanks @fabioacp 
Both way represents the <ESC> characters, but I should have used a single pattern instead of the alias.
I normalise them to use `\e` for both variables.

Reply:
no worries. Great job mate!